rclcpp/src/rclcpp/parameter_client.cpp

```diff
@@ -401,12 +401,12 @@ SyncParametersClient::SyncParametersClient(
 {
   async_parameters_client_ =
     std::make_shared<AsyncParametersClient>(
-    node_base_interface,
-    node_topics_interface,
-    node_graph_interface,
-    node_services_interface,
-    remote_node_name,
-    qos_profile);
+      node_base_interface,
+      node_topics_interface,
+      node_graph_interface,
+      node_services_interface,
+      remote_node_name,
+      qos_profile);
```

Comment:
@rotu this is a clear improvement IMHO

Reply:
Yeah. It's weird that this wasn't the case before. `uncrustify` must be pretending there's an implicit parenthesis on the previous line and aligning to it.